tests/end_to_end/test_tables.py

```diff
@@ -0,0 +1,83 @@
+import pytest  # noqa: E401
```

Comment:
I _think_ this is considered best-practice nowadays; pytest modules should begin with import pytest, even if unused.